### ISSUE TITLE
Training on Custom Dataset

### ISSUE THREAD

**Issue by cccvvvvv** [HuangJunJie2017/BEVDet#198]
First, thank you for sharing good research!

I wanted to learn BEVDET for the custom data I acquired. We have confirmed that the train progresses normally, but the value of the map is 0 in the validation set. In this case, I would like to contact you a few things.

1. The size of the input image in Custom Data is 1920, 1080, unlike Nuscenes. Should I change input_size separately in this case?

2. Can I know how many data I have to learn if I learn with a custom data acquired by 6 Camera like Nuscenes?

3. Or when learning BEVDET with Custom Data, Are there hyperparameters related to Training?

**Comment by HuangJunJie2017** [comment]
@cccvvvvv visualize the predicted result (train/val sets) to see if it is normal.
